lib/spack/spack/modules/common.py

```diff
@@ -269,13 +269,39 @@ def read_module_indices():
 module_type_to_indices = read_module_indices()
```

Comment:
I would like to keep any work like this out of module scope, so we don't continue to increase Spack startup latency.

Reply:
Should be handled now: module indices are computed lazily in `spack.store`